1596819893474	Sync.LogManager	DEBUG	Flushing file log
1596819893478	Sync.LogManager	DEBUG	Log cleanup threshold time: 1595955893478
1596819893483	Sync.LogManager	DEBUG	Done deleting files.
1596819894468	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1596819895667	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1596819897132	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1596819897132	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1596819897133	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1596819898132	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1596819900350	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1596819900350	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1596819900351	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1596819900351	Sync.Service	DEBUG	User-Agent: Firefox/68.11.0 (Linux x86_64) FxSync/1.70.0.20200801212355.desktop
1596819900351	Sync.Service	INFO	Starting sync at 2020-08-07 13:05:00 in browser session FnLGpA2cFM9y
1596819900351	Sync.Service	DEBUG	In sync: should login.
1596819900351	Sync.Service	INFO	User logged in successfully - verifying login.
1596819900352	Sync.BrowserIDManager	DEBUG	unlockAndVerifyAuthState has no user
1596819900352	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1596819900352	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1596819900352	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1596819900352	Sync.ErrorHandler	ERROR	Sync encountered a login error
1596819900352	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1596819900352	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1596819900352	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.js:984:15) JS Stack trace: onNotify@service.js:984:15
1596819900353	Sync.Service	DEBUG	Not syncing: login returned false.
